docs/content/core/volumes.md

```diff
@@ -7,35 +7,44 @@ aliases:
 
 ## Quick Overview of the Volumes Options
 
-If option is marked Default, then you do not have to do anything for this option to be used.
-
-If `DOCKSAL_VOLUMES` option is not Default, then you need to enable it manually to use it.
+If `DOCKSAL_VOLUMES` option is not marked as default, then you need to enable it manually to use it.
 All non-automatic options are enabled by placing `DOCKSAL_VOLUMES="<value from volumes column>"` into the respective
```

Comment:
Should automatic be changed to default here too?

Reply:
Thanks, will change now